packages/jellyfish-transaction/src/script/dftx/dftx_icxorderbook.ts

```diff
@@ -46,6 +46,38 @@ export class CICXCreateOrder extends ComposableBuffer<ICXCreateOrder> {
   }
 }
 
+/**
+ * ICX MakeOffer DeFi Transaction
+ */
+export interface ICXMakeOffer {
+  orderTx: string // ---------------| 32 bytes, txid for which order is the offer
+  amount: BigNumber // -------------| 8 bytes unsigned, amount of asset to swap
+  ownerAddress: Script // ----------| n = VarUInt{1-9 bytes}, + n bytes, address for DFI token for fees, and in case of BTC/DFC order for DFC asset
+  receivePubkey?: string // --------| n = VarUInt{1-9 bytes}, 0x00 (default when undefined) | 0x21 (for COMPRESSED_PUBLIC_KEY_SIZE) | 0x41 (for PUBLIC_KEY_SIZE) + n bytes. See implementation at https://github.com/DeFiCh/ain/blob/ff53dcee23db2ffe0da9b147a0a53956f4e7ee31/src/pubkey.h#L57, address or BTC pubkey in case of DFC/BTC order
+  expiry: number // ----------------| 4 bytes unsigned, when the offer exipres in number of blocks
+  takerFee: BigNumber // -----------| 8 bytes unsigned
```

Comment:
takerFee is not unsigned
```suggestion
  takerFee: BigNumber // -----------| 8 bytes signed
```